docs/streams/developer-guide/app-reset-tool.html

```diff
@@ -84,9 +84,11 @@ <h2>Step 1: Run the application reset tool<a class="headerlink" href="#step-1-ru
 ---------------------                 -----------
 * --application-id &lt;String: id&gt;       The Kafka Streams application ID
                                         (application.id).
---bootstrap-servers &lt;String: urls&gt;    Comma-separated list of broker urls with
-                                        format: HOST1:PORT1,HOST2:PORT2
-                                        (default: localhost:9092)
+--bootstrap-server [String: Server(s)  The server(s) to use for bootstrapping.
```

Comment:
Can we keep the formatting consistent with the other options here, using `<...>` instead of `[...]`?

Reply:
Fixed. Thanks.